app/controllers/projects_controller.rb

```diff
@@ -545,17 +545,39 @@ def validate_sample_names
   def create_new_user_random_password(name, email)
     Rails.logger.info("Going to create new user via project sharing: #{email}")
     user_params_with_password = { email: email, name: name }
-    random_password = SecureRandom.hex(10)
+    random_password = SecureRandom.base64(10)
```

Comment:
This is to satisfy a stricter password complexity policy

Reply:
I would say a random base64 is not a guarantee that will get a combination of all required characters (which I believe is at least one of each: uppercase/lowercase/digit/special_character).
This random base64 could eventually only contains number and letters and fail to pass. I believe we should either use a library to generate these bogus initial passwords (they will never be used, correct?) or append at least one character of each class to this password (something like `SecureRandom.base64(10) + "aA1!"`)

Reply:
We have it as "3 out of 4 of lowercase/uppercase/digits/special characters" (and that's the highest level they have). But you're right that it could still fail. Will update 👍

Reply:
updated 👍